src/styles/blocks/index.scss

```diff
@@ -0,0 +1,59 @@
+@import '../utils/variables';
+
+body {
+  margin: 0;
+  padding-top: 25px;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  width: $col-count * $cell-width + $gap * $col-count;
```

Comment:
Ensure that the variables `$col-count`, `$cell-width`, and `$gap` are defined in the imported `variables` file. Otherwise, this calculation will not work as expected.